Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM docker.io/golang:alpine as builder
```

Comment:
Why do we need this?

Reply:
That's so we can make the deployed image more lightweight without extra dependencies

Reply:
No I meant why do we need this dockerfile?

Reply:
We need a docker compose setup to run web3 proxy with document server as requested in this [ticket](https://github.com/freeflowuniverse/aydo/issues/2).